### ISSUE TITLE
Expected performance

### ISSUE THREAD

**Issue by karandwivedi42** [DeepVoltaire/AutoAugment#1]
Did you run full training with the transformations?

**Comment by DeepVoltaire** [comment]
I am in the process of repeating the experiments on the Fine Grained Visual Classification datasets (section 4.4 in the paper) with and without the ImageNet-Policy. It looks like it is working as expected and can boost your performance on image classification tasks with natural images by a significant amount. When finished I will write a blog post about it and include the code here.

I am currently not planning to repeat the CIFAR10/100, SVHN, ImageNet results as I am more interested in the transfer learning aspect on new datasets. For CIFAR10/100 you would also first need an implementation of ShakeDrop.

**Comment by karandwivedi42** [comment]
Thanks! That's also very interesting because they reach high accuracy without Imagenet pretraining!
